java/connector-node/risingwave-sink-dynamic/src/main/java/com/risingwave/connector/sinkwriter/SinkWriterAdapt.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 RisingWave Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.risingwave.connector.sinkwriter;
+
+import org.apache.flink.table.data.RowData;
+
+public interface SinkWriterAdapt<CommT> {
```

Comment:
What does this `CommT` mean?

Reply:
The type of information needed to commit data staged by the sink. This is to ensure consistency between what is created and what is passed during the commit.

Reply:
But the generic type is not used in any method, so it is same no matter we have this generic type or not?